### PR TITLE
Resubmitting Genoanime Presence

### PR DESCRIPTION
**Pull request by Genoanime**
image

I have updated the presence and removed contributor from meta file as recommended. Please let me know if I need to fix anything else.

### CODE REVIEW

**Comment by Genoanime** [comment]
I thought i squashed the commits but  im still seeing them should i close and resubmit or its fine?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Use prettier

File: websites/G/GenoAnime/presence.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
+const presence = new Presence({
+  clientId: "810203651317432351"
+}),
+      
+strings = presence.getStrings({
```
Comment (EncryptedDev) [inline websites/G/GenoAnime/presence.ts:5]: indentation

File: websites/G/GenoAnime/presence.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        paused = data.iframe_video.paused;
+      }
+    }
+  );  
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "genoanime",
+    startTimestamp: browsingStamp
+  },
+title = document.title.slice(0, -13); //title of the page
```
Comment (EncryptedDev) [inline websites/G/GenoAnime/presence.ts:43]: more indentation to list a few

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/G/GenoAnime/presence.ts (hunk 2 above)
Comment (Genoanime) [reply websites/G/GenoAnime/presence.ts:43]: will do

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Fix codacy now

**Comment by Genoanime** [comment]
> Fix codacy now

Fixed thank you.

**Comment by EncryptedDev** [comment]
Upload new proof too please - after 40 commits, that could be old 🤣

**Comment by Genoanime** [comment]
Yes sir @EncryptedDev 
Added Proof of presence
image
image
image
image
image

**Comment by EncryptedDev** [comment]
Das all gucci from me! LGTM

**Comment by ririxidev** [comment]
like i said squash commits...

**Pull request closed without merging**